diff -Naur gnome-settings-daemon-3.26.1/data/org.gnome.settings-daemon.plugins.media-keys.gschema.xml.in.in new/data/org.gnome.settings-daemon.plugins.media-keys.gschema.xml.in.in
--- old/data/org.gnome.settings-daemon.plugins.media-keys.gschema.xml.in.in	2017-09-01 14:12:11.000000000 +0200
+++ new/data/org.gnome.settings-daemon.plugins.media-keys.gschema.xml.in.in	2017-10-20 14:12:58.468019469 +0200
@@ -95,6 +95,16 @@
       <_summary>Volume up</_summary>
       <_description>Binding to raise the system volume.</_description>
     </key>
+    <key name="volume-down-precise" type="s">
+      <default>'&lt;Shift&gt;XF86AudioLowerVolume'</default>
+      <_summary>Volume down precise</_summary>
+      <_description>Binding to lower the system volume.</_description>
+    </key>
+    <key name="volume-up-precise" type="s">
+      <default>'&lt;Shift&gt;XF86AudioRaiseVolume'</default>
+      <_summary>Volume up precise</_summary>
+      <_description>Binding to raise the system volume.</_description>
+    </key>
     <key name="screenshot" type="s">
       <default>'Print'</default>
       <_summary>Take a screenshot</_summary>
diff -Naur gnome-settings-daemon-3.26.1/plugins/media-keys/shortcuts-list.h new/plugins/media-keys/shortcuts-list.h
--- old/plugins/media-keys/shortcuts-list.h	2017-09-01 14:12:11.000000000 +0200
+++ new/plugins/media-keys/shortcuts-list.h	2017-10-20 14:08:06.870924500 +0200
@@ -53,8 +53,8 @@
         { MUTE_QUIET_KEY, NULL, N_("Quiet Volume Mute"), "<Alt>XF86AudioMute", SHELL_ACTION_MODE_ALL },
         { VOLUME_DOWN_QUIET_KEY, NULL, N_("Quiet Volume Down"), "<Alt>XF86AudioLowerVolume", SHELL_ACTION_MODE_ALL },
         { VOLUME_UP_QUIET_KEY, NULL, N_("Quiet Volume Up"), "<Alt>XF86AudioRaiseVolume", SHELL_ACTION_MODE_ALL },
-        { VOLUME_DOWN_PRECISE_KEY, NULL, N_("Precise Volume Down"), "<Shift>XF86AudioLowerVolume", SHELL_ACTION_MODE_ALL },
-        { VOLUME_UP_PRECISE_KEY, NULL, N_("Precise Volume Up"), "<Shift>XF86AudioRaiseVolume", SHELL_ACTION_MODE_ALL },
+        { VOLUME_DOWN_PRECISE_KEY, "volume-down-precise", N_("Precise Volume Down"), NULL, SHELL_ACTION_MODE_ALL },
+        { VOLUME_UP_PRECISE_KEY, "volume-up-precise", N_("Precise Volume Up"), NULL, SHELL_ACTION_MODE_ALL },
         { LOGOUT_KEY, "logout", NULL, NULL, GSD_ACTION_MODE_LAUNCHER },
         { EJECT_KEY, "eject", NULL, NULL, SHELL_ACTION_MODE_ALL },
         { HOME_KEY, "home", NULL, NULL, GSD_ACTION_MODE_LAUNCHER },
